### PR TITLE
Fixes #1571 (TSD download button)

### LINKED ISSUE #1571: add link to download TSD
TSD = Tech Support Document

### PR DESCRIPTION
**Pull request by TomNUSDS**

### CODE REVIEW

File: client/compiled-lang/en.json [hunk 1]
```diff
@@ -2575,20 +2575,26 @@
       "value": "Download shapefile"
     }
   ],
+  "methodology.page.downloadPacket.button3.text": [
+    {
+      "type": 0,
+      "value": "Download technical support document"
```
Comment (esfoobar-usds) [inline client/compiled-lang/en.json:2581]: Is this not in i18n'd?

**Review by esfoobar-usds — APPROVED** [review]
🍏 -- one small comment

File: client/compiled-lang/en.json (hunk 1 above)
Comment (TomNUSDS) [reply client/compiled-lang/en.json:2581]: It should be. I ran all the scripts and go no errors. Can you point me to where you think it should be to help me out?

Comment (esfoobar-usds) [reply client/compiled-lang/en.json:2581]: I meant, the button text will be translated if/when we have the `es.json`?

Comment (TomNUSDS) [reply client/compiled-lang/en.json:2581]: Yup. EVERYTHING should be in the translation framework.

**Pull request merged**